cpp/src/vns/plan.hpp

```diff
@@ -42,12 +42,14 @@ struct Plan {
     TimeWindow time_window;
     Trajectories core;
     shared_ptr<FireData> firedata;
-    vector<Point3dTimeWindow> possible_observations;
+    vector<PointTimeWindow> possible_observations;
+    vector<PointTimeWindow> actual_observations;
```

Comment:
What is the purpose of actual_observations ?
I don't see it anywhere except on initialization.

The reason I did not include previously is that keeping track of actual observations must be made incrementally which is quickly error prone and would not probably not buy much computational cost.

Reply:
It is just a bogus declaration that souldn't be there. 

You're right on that, keeping track of this is quite error prone.